mlflow/tracking/client.py

```diff
@@ -759,7 +759,7 @@ def log_param(self, run_id: str, key: str, value: Any) -> Any:
                     All backend stores support keys up to length 250, but some may
                     support larger keys.
         :param value: Parameter value (string, but will be string-ified if not).
-                      All backend stores support values up to length 500, but some
+                      All backend stores support values up to length 6000, but some
                       may support larger values.
```

Comment:
`All backend stores support values up to length xxx` sound strange to me. For example, you could create your own MLflow service in Go where the maximum value length is 10.

Reply:
Can we add `built-in` after `All`?